app/i18n/en/main.json

```diff
@@ -72,7 +72,7 @@
     "introTitle": "Add and fact check a new video",
     "intro": "CaptainFact is a collaborative fact-checking tool for YouTube videos. People help each others to verify informations in this way: video is added on the websitec statements are extracted from the video, then fact-checkers confirm or refute the statements. All CaptainFact's extension users will see our verifications directly on YouTube videos.",
     "seeExtension": "Learn more about extension",
-    "intro2": "You can add a new video (if you are a member with at least 75 reputation pts) or you can contact us:",
+    "intro2": "You can add a new video (if you are a member with at least {{requiredPoints}} reputation pts) or you can contact us:",
```

Comment:
Good call turning this one into a variable

Reply:
Sure, the scout rule :).